### ISSUE TITLE
Common IDE, use Codespace?

### ISSUE THREAD

**Issue by MOJOliciousFTW** [MOJOliciousFTW/workshop-github-2024#7]
Should we set up devcontainer and use Codespaces? Some other way of using a common IDE?

**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I want to help out but first need to set up my PC.

**Describe the solution you'd like**
A devcontainer.json with VS Code extensions for spelling and the like. See #6.

**Describe alternatives you've considered**
Pressing "." while browsing repo. Does open VC Code in browser with possibillity to add extensions, but not a terminal. Terminal maybe usefull when working on workflows.

**Additional context**

**Comment by Pofstad** [comment]
Is it possible to give every user access to Github Copilot when using Github Codespace?

**Comment by MOJOliciousFTW** [comment]
> Is it possible to give every user access to Github Copilot when using Github Codespace?

no. GitHub Codespace (or any VS Code setup) requires personal login to Copilot.

Login is handled autmatically. If you have logged in (which you have when using codespace) Copilot either works or notifies that you don't have Copilot.

**Comment by MOJOliciousFTW** [comment]
from a different angle: Copilot is payed for per user, not per repo.

Would be cool if possible, to supply Copilot per repo just for work on repo, a nice incentive to work on repo maybe.
